Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -442,8 +444,9 @@ def relative_glob(dir, pattern)
     File.directory?(dir) ? Dir.chdir(dir) { Dir[pattern] } : []
   end
 
-  def cpuid_instruction?(file, objdump = "objdump")
-    @instruction_column_index ||= {}
+  sig { params(file: T.any(Pathname, String), objdump: Pathname).returns(T::Boolean) }
+  def cpuid_instruction?(file, objdump)
+    @instruction_column_index ||= T.let({}, T.nilable(T::Hash[String, Integer]))
```

Comment:
```suggestion
    @instruction_column_index ||= T.let({}, T.nilable(T::Hash[Pathname, Integer]))
```
Sorry, I may have left you a bad suggestion here previously 🙃

Reply:
My fault for applying it with blind trust, I think!